### ISSUE TITLE
Show blocked elements instead if just blocked ads on icon.

### ISSUE THREAD

**Issue by nextgenthemes** [dhowe/AdNauseam#1277]
### Describe the issue
I am coming from uBlock origin and I am used to see if there are currently **any** blocked elements on the page. I need that when debugging sites ... The issue with AdNauseam is that it hides that original uBlock stats behind **two** mouse-clicks and I only now discovered this. I always thought when it shows a `0` it means no blocked elements, when in fact my privacy filters and stuff are indeed blocking elements but its not visible in the blink of an eye.

Please make it a option just to pass the blocked items to the icon counter instead if just the blocked (or clicked?) ads.

### One or more specific URLs where the issue occurs
For example on this very Github page, there are actually 3 things blocked one of them is google analytics but its not visible to me until I bring up that original uBlock info window.

### Screenshot in which the issue can be seen
Let me know if you need one, I think you understand.

### Your settings

- OS/version: Ubuntu 17.04
- Browser/version: Firefox 57
- AdNauseam version: v3.4.402
- Other extensions you have installed: nothing relevant to this

**Comment by dhowe** [comment]
Yes, I've thought about this before, and considered adding two numbers to the badge, but decided this was two confusing. Are you suggesting a preference where you pick whether to show ads or blocks? And what about hidden elements ?

**Comment by nextgenthemes** [comment]
Option to show blocked elements instead would be great, but I personally am not opposed to two counters.

Newer thought about hidden elements, uBlock does only show blocked i guess? Not opposed to that either (as a option) but I was mainly just looking for what I am used to from uBlock.

**Comment by dhowe** [comment]
Agreed -- add option toggle (under 'interface') to show blocks instead of ads, default=unchecked

**Comment by cqx931** [comment]
Please check https://github.com/dhowe/AdNauseam/pull/1279.
For blocking requests, the badge color is red; For adnauseam ad counts, the badge color is blue.

**Comment by dhowe** [comment]
This was not ready for a PR: see #1281 

On hold until we have an interface plan

**Comment by mneunomne** [comment]
I don't think this is possible to implement in an easy way, also how to quantify "elements" woild be a tricky question since one blocked section can contain several ads, and so on. Closing for now.
